docs/reference/modules/node.asciidoc

```diff
@@ -406,19 +396,18 @@ node.remote_cluster_client: false <9>
 <5> The `node.ml` role is enabled by default.
 <6> The `xpack.ml.enabled` setting is enabled by default.
 <7> Disable the `node.transform` role.
-<8> The `xpack.transform.enabled` setting is enabled by default.
-<9> Disable remote cluster connections (enabled by default).
+<8> Disable remote cluster connections (enabled by default).
 
 [discrete]
 [[transform-node]]
 === [xpack]#{transform-cap} node#
 
 {transform-cap} nodes run {transforms} and handle {transform} API requests.
 
-If you want to use {transforms} in your cluster, you must have
-`xpack.transform.enabled` set to `true` on all master-eligible nodes and all
+If you want to use {transforms} in your cluster, you must not have
+`xpack.transform.enabled` set to `false` on any master-eligible nodes or any
 data nodes. If you want to use {transforms} in clients (including {kib}), it
-must also be enabled on all coordinating nodes. You must also have
+also must not be disabled on any coordinating nodes. You must also have
 `node.transform` set to `true` on at least one node. This is the default
```

Comment:
The "it" in this sentence is a bit ambiguous. I'd say:

To use {transforms} in clients {including {kib}), do not disable this setting on any coordinating nodes.